cmd/rpc-common.go

```diff
@@ -27,6 +27,13 @@ import (
 // 3 seconds is chosen arbitrarily.
 const rpcSkewTimeAllowed = 3 * time.Second
 
+const globalRPCAPIVersionV1 = 1 // Initial version
```

Comment:
can you write this like `iota + 1` style here since we seem to be progressing in this manner?